### DISCUSSION TITLE
Is a three pane view like evernote is in roadmap ?

### DISCUSSION THREAD

**Discussion by arromattic** [TriliumNext/Trilium#4072] [Q&A]
or other chapter styled note apps

**Comment by XXXJetfireXXX** [comment]
You mean like split screen? Yes, you can create as much splits as you want by pressing this button and opening any note you want in a new split.
![image](https://github.com/zadam/trilium/assets/43646178/2d6f7b62-3623-4931-8cb2-5f644d6b5117)
You can also open the same note to view it in several places simultaneously.

**Comment by arromattic** [comment]
I mean't this 
![new-win10-hero](https://github.com/zadam/trilium/assets/133383364/6f5c29f8-6a6f-49d2-8061-293ae65ac65f)
Or this 
![bc9891d07d193a12f6b01edb7eb138686ade3b4e_2_1035x738](https://github.com/zadam/trilium/assets/133383364/b75062fe-1a4c-4fb4-b8db-5997c1b70474)
Three different panes 
one for folders or note books 
one for the last level or notes
one for editor and writing

A easy way to understand is to use obsidian file explorer with some notes and some levels of folders than install alternative tree view plugin and use the second pane which dosen't include preview like evernote but shows last level of notes in a separate pane

**Comment by Nriver** [comment]
Within Trilium, the concept of "folders" is just an ordinary note with a visual representation in the form of a "folder" icon. Distinguishing between notes and folders is unnecessary since all of them are essentially notes.

**Reply by arromattic** [reply]
It makes identifying notes a lot easier

**Comment by zadam** [comment]
To answer the question - no, this is not planned to be implemented.

**Comment by meichthys** [comment]
I moved away from Evernote and Onenote for this exact reason. In those applications there is a limited number or no sub-notes allowed (you cannot write any information in the notebook or sections themselves as they are only containers). You can only write information into a note. Trilium IMO is so much more versatile since every parent note can have it's own content. 

You should be able to get a somewhat closer experience in Trilium if you add the following attribute to your root note and make it inheritable: `#viewType=list`
This way you will see your folder tree in the normal tree area and all sub-notes in your main content area in a list form like those other apps. Clicking on one of the notes in the list will open the note and select the note in the main tree at the left to keep you oriented.

**Reply by arromattic** [reply]
the preview and tag is missing
